### ISSUE TITLE
On editing cells: `ValueError: At based indexing on an non-integer index can only have non-integer indexers`

### ISSUE THREAD

**Issue by phdowling** [quantopian/qgrid#198]
### Environment
* Operating System: Ubuntu
* Python Version: Python 3.6.4 :: Anaconda, Inc.
* How did you install Qgrid: pip
* Python packages: `notebook==5.4.0, qgrid==1.0.3, jupyterlab==0.31.5`
* Jupyter lab packages (if applicable): 
```
JupyterLab v0.31.5
Known labextensions:
   app dir: /home/ubuntu/anaconda3/share/jupyter/lab
@jupyter-widgets/jupyterlab-manager
        @jupyter-widgets/jupyterlab-manager v0.34.0  enabled  OK
qgrid
        qgrid v1.0.3  enabled  OK
```

### Description of Issue
I'm displaying a dataframe, and trying to edit it. I get the following error:
`ValueError: At based indexing on an non-integer index can only have non-integer indexers`

### Reproduction Steps
In a notebook:
```
data = [[0, 28108, 'blah1', 'asd', 6, 0.6093999743461609, True],
 [1, 23277, 'blah2', 'dfg', 47, 0.5604000091552734, True],
 [2, 84668, 'blah3', 'asd', 7, 0.551800012588501, True],
 [3, 171563, 'blah4', 'asd', 8, 0.5310999751091003, True],
 [4, 133630, 'blah5', 'asd', 6, 0.5145000219345093, True]]
df = pandas.DataFrame(data)
grid = qgrid.show_grid(df, grid_options={"autoEdit": True}, show_toolbar=True)
grid
```
Then, try editing any of the cells. In Jupyterlab, I get a popup: `Error occurred while attempting to edit the DataFrame. Check the notebook server logs for more information.`. In the console:
```
[IPKernelApp] ERROR | Error occurred while attempting to edit the DataFrame. Check the notebook server logs for more information.
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/qgrid/grid.py", line 1223, in _handle_qgrid_msg_helper
    old_value = self._df.at[location]
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/pandas/core/indexing.py", line 1868, in __getitem__
    key = self._convert_key(key)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/pandas/core/indexing.py", line 1915, in _convert_key
    raise ValueError("At based indexing on an non-integer "
ValueError: At based indexing on an non-integer index can only have non-integer indexers
```
### What steps have you taken to resolve this already?
Workaround for now was to manually set my_qgrid._df, but this seems hacky and unlikely to work correctly.

**Comment by TimShawver** [comment]
That not good...looks like a new issue with editing columns that have an unnamed index column in version 1.0.3.  I'll have a new release shortly but in the meantime you can get around this by downgrading to version 1.0.2:
```
pip uninstall qgrid
pip install qgrid==1.0.2
```
Thanks for the report.

**Comment by TimShawver** [comment]
This is fixed in version 1.0.5 which is now available on pypi.  It will be available on conda shortly...the build is still in progress.

**Comment by TimShawver** [comment]
Closing because this is fixed.
